### PR TITLE
machines: fix distance between the chart elements in VM Usage tab

### PR DESCRIPTION
**Pull request by KKoukiou**
Nested PF4 tables have issues, we should not follow this pattern.
Replace tables with flex for creating the layout of this component.

Related to https://bugzilla.redhat.com/show_bug.cgi?id=1873931

### CODE REVIEW

**Comment by KKoukiou** [comment]
This broke some tests, reposting.

**Comment by marusak** [comment]
Tests still not happy :(

File: pkg/machines/components/storagePools/storagePoolList.jsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ export class StoragePoolList extends React.Component {
                     <BreadcrumbItem to='#'>
                         {_("Virtual Machines")}
                     </BreadcrumbItem>
-                    <BreadcrumbItem isActive>
+                    <BreadcrumbItem to="#" isActive>
```
Comment (marusak) [inline pkg/machines/components/storagePools/storagePoolList.jsx:49]: This seems unrelated and also wrong. when I now click on `Storage Pools` it takes me to `Virtual Machines`. Before it was not clickable which made more sense.

**Review by marusak — CHANGES_REQUESTED** [review]
I think there is unrelated (and likely wrong) change. Can you please elaborate why it is there?

File: pkg/machines/components/storagePools/storagePoolList.jsx (hunk 1 above)
Comment (KKoukiou) [reply pkg/machines/components/storagePools/storagePoolList.jsx]: Oh sorry, that was not meant to be here.

**Review by marusak — APPROVED** [review]
Thanks, looks good now!
